yt_dlp/extractor/bilibili.py

```diff
@@ -892,19 +892,28 @@ def _get_subtitles(self, *, ep_id=None, aid=None):
                 'aid': aid,
             })) or {}
         subtitles = {}
-        for sub in sub_json.get('subtitles') or []:
-            sub_url = sub.get('url')
-            if not sub_url:
-                continue
-            sub_data = self._download_json(
-                sub_url, ep_id or aid, errnote='Unable to download subtitles', fatal=False,
-                note='Downloading subtitles%s' % f' for {sub["lang"]}' if sub.get('lang') else '')
-            if not sub_data:
-                continue
-            subtitles.setdefault(sub.get('lang_key', 'en'), []).append({
-                'ext': 'srt',
-                'data': self.json2srt(sub_data)
-            })
+
+        for sub in sub_json.get('video_subtitle') or []:
```

Comment:
This is not non fatal. You are using `.get`, which, if its not a dict, will fail. Use
```suggestion
        for sub in traverse_obj(sub_json, ('video_subtitle', ..., {dict})):
```

Reply:
This is done.

Reply:
You did not use `for sub in traverse_obj(sub_json, ('video_subtitle', ..., {dict})):`. This is now still fatal.

Reply:
corrected it.